src/mpyl/projects/versioning.py

```diff
@@ -112,6 +113,22 @@ def upgrade(self, previous_dict: ordereddict) -> ordereddict:
         return previous_dict
 
 
+class ProjectUpgraderOneFour19(Upgrader):
```

Comment:
I couldn't find any documentation about this process so I had to dig through the project history to find a similar PR and "blindly" apply a similar set of changes. Please give it a careful review as it's likely missing something important.

Reply:
I actually found the same problem when I was doing schema change recently and we defined this ticket: https://vandebron.atlassian.net/browse/GUILD-773

Reply:
@ptab maybe you could add the documentation that you found lacking?